### PR TITLE
CEAEDFOCCT.SALOME version 9.8.0

### PR DESCRIPTION
**Pull request by mircopergreffi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56725)

### CODE REVIEW

**Comment by Trenly** [comment]
@ryfu-msft 
```
2022-04-03 18:40:08.258   Message box (Abort/Retry/Ignore):
                          C:\SALOME-9.8.0\W64\qt\bin\linguist.exe
                          
                          An error occurred while trying to copy a file:
                          There is not enough space on the disk.
```
image

File: manifests/c/CEAEDFOCCT/SALOME/9.8.0/CEAEDFOCCT.SALOME.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: CEAEDFOCCT.SALOME
+PackageVersion: 9.8.0
+PackageLocale: en-US
+Publisher: CEA EDF OCCT
+PackageName: SALOME
+License: LGPL-2.1-or-later
+ShortDescription: SALOME Setup
```
Comment (Trenly) [inline manifests/c/CEAEDFOCCT/SALOME/9.8.0/CEAEDFOCCT.SALOME.locale.en-US.yaml:10]: Can you please provide a better description of what SALOME is?

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/CEAEDFOCCT/SALOME/9.8.0/CEAEDFOCCT.SALOME.locale.en-US.yaml (hunk 1 above)
Comment (mircopergreffi) [reply manifests/c/CEAEDFOCCT/SALOME/9.8.0/CEAEDFOCCT.SALOME.locale.en-US.yaml]: I have updated the short description

**Comment by mircopergreffi** [comment]
I have realized that windows warns me that the publisher of the file is unknown, perhaps this is why it fails.
I think the executable is not signed.

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
> I have realized that windows warns me that the publisher of the file is unknown, perhaps this is why it fails.

Azure Pipelines is running out of space when installing this so it cannot continue.

**Pull request closed without merging**